tutorial_code/EXAMPLES_LICENSE

```diff
@@ -0,0 +1,14 @@
+Zero-Clause BSD License
```

Comment:
Maybe this should just be LICENSE given that nothing else is here?

Reply:
Renamed to LICENSE_BSD0 (similar to what we have in the docs folder).